app/src/main/java/com/google/jetpackcamera/ui/JcaApp.kt

```diff
@@ -30,8 +40,23 @@ fun JcaApp() {
         rememberPermissionState(permission = Manifest.permission.CAMERA)
 
     if (permissionState.status.isGranted) {
-        PreviewScreen()
+        JetpackCameraNavHost(modifier = Modifier.fillMaxSize())
```

Comment:
Related to name of `JcaApp`: I thought this was actually a data or domain class
Maybe we should call the main classes hosting composables postfixed with `Screen` ?

Reply:
Followed the same convention from NiaApp, we can discuss this and see what's more appropriate.